README.md

```diff
@@ -103,9 +103,9 @@ The `saml20-idp-remote.php` file should look like :
 
    //...
    //...
-$metadata['https://sso.phplist.com:8443/auth/realms/master'] = [
-    'SingleSignOnService'  => 'https://sso.phplist.com:8443/auth/realms/master/protocol/saml',
-    'SingleLogoutService'  => 'https://sso.phplist.com:8443/auth/realms/master/protocol/saml',
+$metadata['https://sso.phplist.com:8443/realms/master'] = [
+    'SingleSignOnService'  => 'https://sso.phplist.com:8443/realms/master/protocol/saml',
```

Comment:
this needs to become configurable, either in the config.php file or with the config page

Reply:
in fact all of the settings of simplesaml need to become configurable.

I think for now, the easiest will be to put that in the config file and document how to do it, but it may be nice to make it work with the config page, although you'd need to be careful not to get locked out when you make an error in the config